node-template/runtime/build.rs

```diff
@@ -19,7 +19,11 @@ use wasm_builder_runner::{build_current_project_with_rustflags, WasmBuilderSourc
 fn main() {
 	build_current_project_with_rustflags(
 		"wasm_binary.rs",
-		WasmBuilderSource::Crates("1.0.6"),
+		// Note: replace with `WasmBuilderSource::Crates("1.0.6")` if you copy-paste this template.
+		WasmBuilderSource::CratesOrPath {
```

Comment:
Please up the version of wasm-builder to `1.0.7`, publish it to crates.io and then restore the old behavior here.